Pod/Classes/URBNDataSourceAdapter.m

```diff
@@ -276,7 +276,7 @@ - (void)registerCellClass:(Class)cellClass withIdentifier:(NSString *)identifier
     ASSERT_TRUE(self.tableView || self.collectionView);
     
     identifier = identifier?:NSStringFromClass(cellClass);
-    UINib* nib = [self nibWithName:identifier];
+    UINib* nib = [self nibWithName:NSStringFromClass(cellClass)];
```

Comment:
it won't work anyway. if a nib name doesn't match the class name you have to load the nib from a non-nib initializer.  this matches the supplementary view logic